sites/cnfpt/src/backend/locale/fr/LC_MESSAGES/django.po

```diff
@@ -21,32 +21,52 @@ msgstr ""
 msgid "Unsorted uploads are not allowed."
 msgstr "Les téléchargements non classés sont interdits"
 
-#: cnfpt/settings.py:210
+#: cnfpt/settings.py:213
+msgid "Dashboard"
+msgstr ""
+
+#: cnfpt/settings.py:214
+msgid "{base_url:s}/dashboard"
```

Comment:
ditto

Reply:
On OpenEdX this url does not change according to the language.